submissions/best-of-its-genre-part-2-nanogamer7/influence.md

```diff
@@ -0,0 +1,62 @@
+## The Best of Its Genre – Influence of a Game
+
+A few months ago in [issue #34](https://alt-f4.blog/ALTF4-34/) I tried to find the origins of Factorio, and with it the origins of the automation genre. But since Wube pioneered this genre in 2013, many games adapted and embraced it, like [Satisfactory](https://store.steampowered.com/app/526870/Satisfactory/) and [Dyson Sphere Program](https://store.steampowered.com/app/1366540/Dyson_Sphere_Program/), but also games more focused on survival like Astroneer, later versions of Minecraft and more recent mods for it. In this article I want to explore what became of the automation genre, and how automation games are differentiated to survival games with some automation.
+
+{screenshot(s) cause images are good to break up text}
+
+### Part 1: The Big Three
```

Comment:
If you write "Part 1" it sounds like we need to get ready for a multiple sections part. I think it will be better with simply "The Big Three"

Reply:
Tried to give the structure my own [fancier word for] spin. If it doesn't work out for @ClaudeMetz either we can remove it, but for now I'll leave it in